### ISSUE TITLE
Customizations & AccountNotFoundException

### ISSUE THREAD

**Issue by Menahem1** [aws-ia/terraform-aws-control_tower_account_factory#289]
**AFT Version:** 1.7.0

**Bug Description**
When applying Step Functions (aft-invoke-customizations) with all account specified i encountered the following error

`"type": "all"`

```
{
    "time_stamp": "2023-01-03 10:45:09,527",
    "log_level": "ERROR",
    "log_message": {
        "FILE": "aft_customizations_identify_targets.py",
        "METHOD": "lambda_handler",
        "EXCEPTION": "An error occurred (AccountNotFoundException) when calling the DescribeAccount operation: You specified an account that doesn't exist."
    }
}
Traceback (most recent call last):
  File "/var/task/aft_customizations_identify_targets.py", line 64, in lambda_handler
    account_email = orgs_agent.get_account_email_from_id(account_id)
  File "/opt/python/lib/python3.8/site-packages/aft_common/organizations.py", line 247, in get_account_email_from_id
    response: DescribeAccountResponseTypeDef = self.orgs_client.describe_account(
  File "/opt/python/lib/python3.8/site-packages/botocore/client.py", line 508, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/opt/python/lib/python3.8/site-packages/botocore/client.py", line 915, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.errorfactory.AccountNotFoundException: An error occurred (AccountNotFoundException) when calling the DescribeAccount operation: You specified an account that doesn't exist.
```

On the log before the error i have the "Getting details for xxxxxxxxxx" but that account is existing
I'm not sure but it seems to be a deactivated account 
how to handle this case ?

Thanks

**Comment by stumins** [comment]
Hi @Menahem1,

Thanks for the report. This could be caused by trying to describe a suspended account - are there any records in the `aft-request-metadata` DynamoDB table for suspended accounts?

**Comment by Menahem1** [comment]
Hi @stumins 
Thanks for your reply !
On` aft-request-metadata` all the account's are in status "active"

**Comment by stumins** [comment]
We just released [AFT 1.8.0](https://github.com/aws-ia/terraform-aws-control_tower_account_factory/releases/tag/1.8.0) which includes a fix to ignore suspended accounts when invoking customizations.

Please upgrade to 1.8.0 and let us know if it resolves this issue.

**Comment by Menahem1** [comment]
Thanks @stumins it's better now !
